collectors/proc.plugin/sys_block_zram.c

```diff
@@ -73,7 +73,7 @@ static inline void init_rrd(const char *name, ZRAM_DEVICE *d, int update_every)
         , RRDSET_TYPE_AREA);
     d->rd_compr_data_size = rrddim_add(d->st_usage, "compressed", NULL, 1, 1024 * 1024, RRD_ALGORITHM_ABSOLUTE);
     d->rd_metadata_size = rrddim_add(d->st_usage, "metadata", NULL, 1, 1024 * 1024, RRD_ALGORITHM_ABSOLUTE);
-    rrdlabels_add(d->st_usage->rrdlabels, "device", name, RRDLABEL_SRC_AUTO);
+    rrdlabels_add(d->st_usage->rrdlabels, "device", name,RRDLABEL_SRC_AUTO);
```

Comment:
In normal situation we are using a  `comma` + `space`, but here I see that `space` was removed, is there a specific reason for this?
Line 111 also has this change.

Reply:
will fix